app/javascript/src/Policies/styles/policies.scss

```diff
@@ -15,12 +15,6 @@ $error-color: #c00;
   line-height: 1.5;
   padding-bottom: 2rem;
 
-  .PolicyChain, .PolicyRegistryList, .PolicyConfiguration {
```

Comment:
I think the partial will continue to be rendered in the integration form for non-APIAP accounts. Won't this change affect the way those users see the Policies section in the page?

Reply:
Even for APIAP-viewers some minor fixes need to be done:

![Screen Shot 2019-09-25 at 1 17 30 PM](https://user-images.githubusercontent.com/1842261/65596342-f16c3600-df96-11e9-8eae-e9e8cfc56c3d.png)